### PR TITLE
update readme nov 2024

### PR DESCRIPTION
**Pull request by leonardoalt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,28 +23,32 @@ It has two main components:
   
 Both frontend and backend are highly flexible.
 
-As an example, *powdr* contains a frontend that enables you to write code in (no-std) Rust,
-which is compiled to RISCV, then to powdr-asm and finally to powdr-PIL.
+The [powdrVM](https://docs.powdr.org/quick_start_vm.html) is a zkVM
+implementation that builds on *powdr*.
+Users can write code in std Rust which is compiled to RISCV,
+then to powdr-asm and finally to powdr-PIL.
```
Comment (pacheco) [inline README.md:29]: there's both `powdr-PIL` and `powdr-pil` in the text

File: README.md [hunk 2]
```diff
@@ -23,28 +23,32 @@ It has two main components:
...
 All stages are fully automatic, which means you do not need to write any
 additional code for witness generation besides your Rust code. All witnesses
 are automatically inferred from the constraints. Since the witnesses are
 inferred, *powdr* can ensure that the system is not underconstrained, i.e.
 there are no additional unwanted witnesses.
 
-All artifacts from the compilation pipeline are human-readable. You
-can inspect the RISCV assembly files, the powdr-asm IR, and the compiled PIL file.
+All artifacts from the compilation pipeline are human-readable.
+You can inspect the the powdr-asm IR, the compiled PIL file,
```
Comment (ShuangWu121) [inline README.md:50]:
```suggestion
You can inspect the powdr-asm IR, the compiled PIL file,
```

File: README.md [hunk 3]
```diff
@@ -23,28 +23,32 @@ It has two main components:
...
+- [Stwo](https://github.com/starkware-libs/stwo/) (under development)
 - eSTARK, using
     - [Eigen's starky](https://github.com/0xEigenLabs/eigen-zkvm/)
     - [our fork of Polygon's zkevm-prover](https://github.com/powdr-labs/zkevm-prover)
-- Plonky3 (ongoing work): https://github.com/powdr-labs/powdr/pull/1158
 
 All stages are fully automatic, which means you do not need to write any
 additional code for witness generation besides your Rust code. All witnesses
 are automatically inferred from the constraints. Since the witnesses are
 inferred, *powdr* can ensure that the system is not underconstrained, i.e.
```
Comment (ShuangWu121) [inline README.md:46]:
```suggestion
inferred, *powdr* can ensure that the system is not underconstrained, i.e.,
```

File: README.md (hunk 1 above)
Comment (leonardoalt) [reply README.md:29]: not in my changes :grimacing:

Comment (leonardoalt) [reply README.md:29]: done

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (leonardoalt) [reply README.md:50]: done

Comment (leonardoalt) [reply README.md:50]: done

File: README.md (hunk 3 above)
Comment (leonardoalt) [reply README.md:46]: done

**Review by pacheco — APPROVED** [review]

**Review by ShuangWu121 — APPROVED** [review]

**Pull request merged**